src/ocrd_network/process_helpers.py

```diff
@@ -28,23 +30,58 @@ def invoke_processor(
 
     workspace = get_ocrd_workspace_instance(mets_path=abs_path_to_mets, mets_server_url=mets_server_url)
     if processor_class:
-        ctx_mgr = redirect_stderr_and_stdout_to_file(log_filename) if log_filename else nullcontext()
-        with ctx_mgr:
-            initLogging(force_reinit=True)
-            try:
-                run_processor(
-                    processorClass=processor_class,
-                    workspace=workspace,
-                    input_file_grp=input_file_grps_str,
-                    output_file_grp=output_file_grps_str,
-                    page_id=page_id,
-                    parameter=parameters,
-                    instance_caching=True,
-                    mets_server_url=mets_server_url,
-                    log_level=log_level
-                )
-            except Exception as error:
-                raise RuntimeError(f"Python executable '{processor_class.__dict__}', error: {error}")
+        with NamedTemporaryFile(mode='w') as cfgfile:
+            cfgfile.write("""
+[loggers]
+keys=root,ocrd,ocrd_network,tensorflow,shapely_geos
+[handlers]
+keys=fileHandler
+[formatters]
+keys=defaultFormatter
+[logger_root]
+level=WARNING
+handlers=fileHandler
+[logger_ocrd]
+level=INFO
+handlers=
+qualname=ocrd
+[logger_ocrd_network]
+level=INFO
+handlers=
+qualname=ocrd_network
+[logger_tensorflow]
+level=ERROR
+handlers=
+qualname=tensorflow
+[logger_shapely_geos]
+level=ERROR
+handlers=
+qualname=shapely.geos
+[handler_fileHandler]
+class=FileHandler
+formatter=defaultFormatter
+args=('{log_filename}','a+')
+[formatter_defaultFormatter]
+format=%(asctime)s.%(msecs)03d %(levelname)s %(name)s - %(message)s
+datefmt=%H:%M:%S
+""".format(log_filename=log_filename))
```

Comment:
Could you put that constant string as a variable above the method definition and just use the variable in the method?

Reply:
Or perhaps another time – should I track this as a separate issue?